query-node/substrate-query-framework/index-builder/src/db/helper.ts

```diff
@@ -10,6 +10,9 @@ import { DeepPartial } from 'typeorm';
  * @param entity: DeepPartial<T>
  */
 export function fillRequiredWarthogFields<T>(entity: DeepPartial<T>): DeepPartial<T> {
+  // Modifying an existing entity so do not add warthog fields
+  if (entity.hasOwnProperty('id')) return entity;
```

Comment:
What about the extra fields like `createdAt` ? Do we still need them in this case?

Reply:
This guard is for updating existing db records. We don't need those extra fields but warthog `BaseModel` adds them :(

Reply:
Ah ok. I thought that this check is like 'if the user has provided an ID field in the schema, use it'

Reply:
👍